etc/ci/check_dynamic_symbols.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+ACTUAL_SYMBOLS=$(arm-linux-androideabi-objdump -T target/arm-linux-androideabi/debug/libservo.so | grep "D " | grep "UND" | awk '{ print $NF; }' | tr '\n' ';')
```

Comment:
According to [buildbot/master/master.cfg#L219](https://github.com/cengizIO/saltfs/blob/master/buildbot/master/master.cfg#L219), `libservo.so` is built in debug mode, with default `mach` target path.